### PR TITLE
CORTX-29502: PI-6 OVA Instructions and Release Notes

### PR DESCRIPTION
**Pull request by venkuppu-chn**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

-----
[View rendered doc/ova/2.0.0/PI-6/CHANGELOG.md](https://github.com/venkuppu-chn/cortx/blob/main/doc/ova/2.0.0/PI-6/CHANGELOG.md)
[View rendered doc/ova/2.0.0/PI-6/S3_IO_Operations.md](https://github.com/venkuppu-chn/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md)

### CODE REVIEW

**Review by shailesh-vaidya — COMMENTED** [review]
- Do we need to add the below steps to use `s3.seagate.com` for IO operations from https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/S3_IO_Operations.md#procedure 

Edit /etc/hosts file and copy the IP of service data pod i.e. cortx-data-pod on same OVA image and add following entry if it does not exist
```
<IP_Address_of_cortx-data-pod> s3.seagate.com iam.seagate.com
```

- Do we need to add steps to test IO operations from outside OVA VM

**Comment by venkuppu-chn** [comment]
> * Do we need to add the below steps to use `s3.seagate.com` for IO operations from https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/S3_IO_Operations.md#procedure
> 
> Edit /etc/hosts file and copy the IP of service data pod i.e. cortx-data-pod on same OVA image and add following entry if it does not exist
> 
> ```
> <IP_Address_of_cortx-data-pod> s3.seagate.com iam.seagate.com
> ```
> 
> * Do we need to add steps to test IO operations from outside OVA VM

@kupranay and @cdeshmukh Do we need to add the steps mentioned by Shailesh? Kindly confirm.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst:1]: @venkuppu-chn, change file path from CORTX_on_Open_Virtual_Appliance_PI-5.rst to CORTX_on_Open_Virtual_Appliance_PI-6.rst

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Comment by kupranay** [comment]
> https://github.com/Seagate/cortx/pull/1476#issuecomment-1082777909

@venkuppu-chn This should not be required

**Comment by johnbent** [comment]
@venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?

**Comment by mukul-seagate11** [comment]
> @venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?

@johnbent, Its because we are waiting for the QA with final validation report so once good, it will be merged

**Comment by johnbent** [comment]
> > @venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?
> 
> @johnbent, Its because we are waiting for the QA with final validation report so once good, it will be merged

Awesome! Thanks. By the way, Greg Kittlison shared an article with me today about best practices for github and it suggested opening pull requests early just like you are doing cool. Nice work!

https://petabridge.com/blog/use-github-professionally/

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst]: Done. Thanks Mukul.

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya @kupranay @cdeshmukh, can you review & approve the instructed updated for OVA

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: Updated OVA image link will be updated once available

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:22]: Update the spec as per new OVA as,

- RAM: 10GB
- Processor: 8

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst]: Done.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:71]: remove Troubleshooting section as not required for now

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Comment by mukul-seagate11** [comment]
Also create a new section for CSM API validation

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:62]: Update the correct URL for S3 IO operations as the link is broken

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: @venkatesh, Update the correct URL for S3 IO operations as the link is broken

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (cdeshmukh) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: https://github.com/Seagate/cortx/releases/tag/2.0.0-585
link has legacy cortx-s3server reference still, should be replaced with cortx-rgw (cc @shailesh-vaidya to confirm that tagging is added for this repo too)

> To clone & checkout from tag 2.0.0-585 where following repos have the CORTX tagging feature:
cortx-s3server

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:62]: What is the correct URL? This link will work once PR is merged if I'm not wrong.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: @mukul-seagate11 What is the correct URL? I believe this link will work once PR is merged.

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: I see as its a newly created file so will be accessible once PR is merged so you can ignore

File: doc/ova/2.0.0/PI-6/CHANGELOG.md
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:13]: Removed.

File: doc/ova/2.0.0/PI-6/CHANGELOG.md
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:43]: Removed the Feature ID.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst]: Done

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: @mukul-seagate11 Please help me to understand about the @chaozhang95 comments.

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: @cdeshmukh, https://github.com/Seagate/cortx/releases/tag/2.0.0-585 visible as old release but the instructions in this page will be updated with new release build i.e. 2.0.0-713.ova in the release page before releasing

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: doc/ova/2.0.0/PI-6/CHANGELOG.md
Comment (cdeshmukh) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:12]: i said merge, keep both tickets in list

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst
Comment (cdeshmukh) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: so page with new release build i.e. 2.0.0-713.ova will not have s3server reference ? if yes, then its fine

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: yes correct, s3server refs are removed in 2.0.0-713.ova

File: doc/ova/2.0.0/PI-6/CHANGELOG.md
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md]: @cdeshmukh, can you review & approve the PR

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**
